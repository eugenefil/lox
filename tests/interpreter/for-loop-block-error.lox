for ch in "foo" {
    x;
}
