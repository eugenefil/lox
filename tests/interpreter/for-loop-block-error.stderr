error: identifier 'x' is not defined
 --> $DIR/for-loop-block-error.lox:2:5
  |
2 |     x;
  |     ^
